fake_useragent/fake.py

```diff
@@ -58,6 +59,14 @@ def __init__(
                 'safe_attrs must be list\\tuple\\set of strings or unicode'
 
         self.safe_attrs = set(safe_attrs)
+        
+        assert isinstance(num_newest_uas, int), \
+            'num_newest_uas must be an integer'
+
+        assert (num_newest_uas > 0), \
```

Comment:
PEP8: Don't use backslash line termination in Python because one whitespace to the right of the backslash breaks the script on a change that is invisible to the reader.  Please use ( ), [ ], or { } instead.